### ISSUE TITLE
bitwarden: outdated electron/package

### ISSUE THREAD

**Issue by surfaceflinger** [NixOS/nixpkgs#262730]
### Describe the bug
It looks like Bitwarden desktop client package is using electron_24. It should be using 25 now(?) https://github.com/bitwarden/clients/blob/master/package.json#L90
Also 2023.9.3 is now out while nixpkgs still is on 2023.9.0.

### Additional context
electron_24 is now EOL and building bitwarden requires allowing insecure packages.

### Notify maintainers

@amarshall @kiwi
And @dotlambda since you touched this month ago and might help in case something breaks :)

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.57-xanmod1, NixOS, 23.11 (Tapir), 23.11.20231020.5672059`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.17.1`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by SuperSandro2000** [comment]
The electron is not outdated, the package just needs an update which will also fix this.

**Comment by SuperSandro2000** [comment]
will be fixed with #260699

**Comment by surfaceflinger** [comment]
epic! thanks
